core/templates/dev/head/services/contextual/device-info.service.ts

```diff
@@ -15,9 +15,6 @@
 /**
  * @fileoverview Service to check if user is on a mobile device.
  */
-
```

Comment:
Why this change?

Reply:
I removed it just to clean the blank lines.